docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device/remote-tensor-api-npu-plugin.rst

```diff
@@ -0,0 +1,136 @@
+.. {#openvino_docs_OV_UG_supported_plugins_NPU_RemoteTensor_API}
+
+Remote Tensor API of NPU Plugin
+===============================
+
+
+.. meta::
+   :description: The Remote Tensor API of NPU plugin in OpenVINO™ supports
+                 interoperability with existing native APIs, such as
+                 Microsoft DirectX, or DMA-BUF System Heap.
+
+
+The NPU plugin implementation of the ``ov::RemoteContext`` and ``ov::RemoteTensor`` interface supports NPU
+pipeline developers who need memory sharing with existing native APIs, such as Microsoft DirectX,
```

Comment:
with other APIs (e.g. OpenCL, Vulkan, DirectX 12) by exporting an NT handle on Windows o DMA-BUF on Linux and passing that pointer as the `shared_buffer` member to the `remote_tensor(..., shared_buffer)` create function.

Reply:
done, thanks!